.github/workflows/on-demand-ocr-soak-test.yml

```diff
@@ -0,0 +1,92 @@
+name: On Demand OCR Soak Test
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: Network to run tests on
+        type: choice
+        options:
+          - "SIMULATED"
+          - "GOERLI"
+          - "OPTIMISM_GOERLI"
+          - "ARBITRUM_GOERLI"
+      fundingPrivateKey:
+        description: Private funding key (Skip for Simulated)
+        required: false
+        type: string
+      wsURL:
+        description: WS URL for the network (Skip for Simulated)
+        required: false
+        type: string
+      httpURL:
+        description: HTTP URL for the network (Skip for Simulated)
+        required: false
+        type: string
+      slackMemberID:
+        description: Notifies test results (Not your @)
+        required: true
+        default: U01A2B2C3D4
+        type: string
+      chainlinkImage:
+        description: Container image location for the Chainlink nodes
+        required: true
+        default: public.ecr.aws/chainlink/chainlink
+      chainlinkVersion:
+        description: Container image version for the Chainlink nodes
+        required: true
+        default: "1.10.0"
+      testDuration:
+        description: Duration of the test (time string)
+        required: false
+        default: 15m
+      chainlinkNodeFunding:
+        description: How much to fund each Chainlink node (in ETH)
+        required: false
+        default: ".001"
+      timeBetweenRounds:
+        description: How long to wait before starting a new round
+        required: false
+        default: 1m
+
+jobs:
+  ocr_soak_test:
+    environment: integration
+    runs-on: ubuntu-latest
+    env:
+      SELECTED_NETWORKS: ${{ inputs.network }}
+      EVM_KEYS: ${{ inputs.fundingPrivateKey }}
+      EVM_URLS: ${{ inputs.wsURL }}
+      EVM_HTTP_URLS: ${{ inputs.httpURL }}
+      SLACK_USER: ${{ inputs.slackMemberID }}
+      SLACK_API_KEY: ${{ secrets.QA_SLACK_API_KEY }}
+      SLACK_CHANNEL: ${{ secrets.QA_SLACK_CHANNEL }}
+      OCR_TEST_DURATION: ${{ inputs.testDuration }}
+      OCR_CHAINLINK_NODE_FUNDING: ${{ inputs.chainlinkNodeFunding }}
+      OCR_TIME_BETWEEN_ROUNDS: ${{ inputs.timeBetweenRounds }}
+      TEST_LOG_LEVEL: debug
+    steps:
+      - name: Collect Metrics
+        id: collect-gha-metrics
+        uses: smartcontractkit/push-gha-metrics-action@v1
+        with:
+          basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
+          hostname: ${{ secrets.GRAFANA_CLOUD_HOST }}
+          this-job-name: ${{ matrix.network }} Smoke Tests
```

Comment:
You aren't using a matrix so this value will be null. You will want to add a name to the job and then put that name here.

Reply:
Thanks for the catch, fixed